teamcity-rest-client-impl/src/test/kotlin/org/jetbrains/teamcity/rest/QueueTest.kt

```diff
@@ -19,7 +19,7 @@ class QueueTest {
 
     @Test
     fun test_kotlin_dev() {
```

Comment:
This test isn't related to kotlin anymore so it should be renamed to avoid confusion.

Reply:
Fixed.